src/prebid.js

```diff
@@ -481,17 +485,26 @@ $$PREBID_GLOBAL$$.clearAuction = function() {
  * @param adUnitCodes
  */
 $$PREBID_GLOBAL$$.requestBids = function ({ bidsBackHandler, timeout, adUnits, adUnitCodes }) {
+  timeout = timeout || $$PREBID_GLOBAL$$.bidderTimeout;
+  adUnits = adUnits || $$PREBID_GLOBAL$$.adUnits;
+
+  // if specific adUnitCodes filter adUnits for those codes
+  if (adUnitCodes && adUnitCodes.length) {
+    adUnits = adUnits.filter(adUnit => adUnitCodes.includes(adUnit.code));
+  }
+
   if (auctionRunning) {
-    utils.logError('Prebid Error: `$$PREBID_GLOBAL$$.requestBids` was called while a previous auction was' +
-      ' still running. Resubmit this request.');
+    bidRequestQueue.push(() => {
+      $$PREBID_GLOBAL$$.requestBids({ bidsBackHandler, timeout, adUnits, adUnitCodes });
```

Comment:
Fix pending, but raised the question: if `adUnits` and `adUnitCodes` are both supplied, is the intended behavior that the ad units passed in will be filtered for codes included in the supplied ad unit codes array?